tests/tests.R

```diff
@@ -0,0 +1,11 @@
+# phoible data tests
+
+library(dplyr)
+library(testthat)
+
+load('../data/phoible.RData')
+
+# Metadata categories should be unique within inventories
+ids <- phoible %>% select(InventoryID) %>% distinct()
+metadata <- phoible %>% select(InventoryID, Glottocode, ISO6393, LanguageName, SpecificDialect, Source) %>% distinct()
+expect_equal(nrow(ids), nrow(metadata))
```

Comment:
missing newline at EOF

Reply:
@drammock - i hate the new line. :)

Reply:
You should set your text editor to put it in automatically when saving the file. :)